internals/webpack/webpack.base.babel.js

```diff
@@ -31,10 +31,19 @@ module.exports = (options) => ({
       loader: 'file-loader',
     }, {
       test: /\.(jpg|png|gif)$/,
-      loaders: [
-        'file-loader',
-        'image-webpack?{progressive:true, optimizationLevel: 7, interlaced: false, pngquant:{quality: "65-90", speed: 4}}',
-      ],
+      loaders: 'file-loader',
+    }, {
+      test: /\.(jpg|png|gif)$/,
```

Comment:
This file pattern match is now listed twice?
This also seems to differ from the [official documentation](https://github.com/tcoopman/image-webpack-loader#usage), is there a reason for this?